### ISSUE TITLE
0.16.0 release

### ISSUE THREAD

**Issue by dcherian** [pydata/xarray#4031]
It'd be nice to issue a release soon. We should decide if this is a minor 0.15.2 or major 0.16.0

Please edit this list as you wish.

**Must-have**

- [x] #3936 multiple dims argmin, argmax

- [x] #4019, #4088; MultiIndex to sparse DataArray bug
- [x] #3922 fix dask array handling in idxmax, idxmin
- [x] #4009, #4173 combine_attrs with open_mfdataset
- [x] #3824 transpose coords (if major release)
- [x] #3926 remove old autocombine
- [x] #4135 nD dask arrays in idxmax/min

**Nice to have**

- [x] #4049, #4094 unstacking merge bug
- [ ] #4022 apply_ufunc meta vectorize
- [ ] #3924 interpolating to coordinates with nans
- [ ] #3925 sel along unindexed 1d coordinates
- [ ] #3993 replace dim with coord in integrate
- [ ] #3594 unit support with pint
- [ ] #4003 mfdataset with zarr
- [x] #3905 length of dataarray reprs
- [x] #3847 error summary for assert_allclose
- [x] #4033 infer_freq
- [x] #3938 plotting with multiindex
- [x] #3816 map_blocks template
- [x] #3976 in-place replacement error message
- [x] #3975 pint dataset
- [x] #4018 netdf3 dtype coercion
- [ ] #3985 groupby multi-index bug (ref discussion below & in the issue)

**Comment by mathause** [comment]
- [ ]  #4075 passing boolean weights to weighted mean

would be nice

**Comment by max-sixty** [comment]
As discussed on the dev call, I'll take this one.

I think we need to do 0.16.0 as we had some backward-incompat changes (I can't remember now what they were; but can hunt them down if anyone otherwise has a preference for 0.15.2?)

**Comment by dcherian** [comment]
Yes we have some breaking changes so definitely 0.16.0

IMO we should do #4184 #4175 and #4163 and then release. #3985  is still waiting for a minimal example so perhaps we should leave that for the next release.

**Comment by max-sixty** [comment]
Agree! I'll keep an eye on those this week.

**Comment by max-sixty** [comment]
Thanks for updating @dcherian . Is it worth waiting for #3985? I'm still not sure exactly what the bug is there. I'd vote to leave it until we have a repro.

I was thinking of sneaking in https://github.com/pydata/xarray/issues/3891 but didn't want to bring it up as probably won't be able to; definitely not worth waiting on me for!

**Comment by dcherian** [comment]
> I'd vote to leave it until we have a repro.

:+1: I'm also not sure what the bug is. In fact it's possible that a bug was fix and a workaround is what's raising the error now :P

Re: #3891 do you mean set `keep_attrs=True` by default?

**Comment by max-sixty** [comment]
> Re: #3891 do you mean set `keep_attrs=True` by default?

Exactly, so this will push to 0.17

**Comment by max-sixty** [comment]
FYI I will do this towards the end of this week or at the latest this weekend

**Comment by dcherian** [comment]
Thanks @max-sixty !
